src/gui/folderman.cpp

```diff
@@ -825,8 +858,9 @@ QString FolderMan::findGoodPathForNewSyncFolder(const QString &basePath, const Q
     {
         QString folder = normalisedPath;
         for (int attempt = 2; attempt <= 100; ++attempt) {
+            // HvR: is this correct?
```

Comment:
What do you mean?

Reply:
Is `FolderMan::findGoodPathForNewSyncFolder` only used for OC10 sync roots or Spaces sync roots? So never for Spaces folders?

Reply:
Its used in both cases.

Reply:
done